pkg/ingestor/parser/slsa/parser_slsa.go

```diff
@@ -191,7 +191,11 @@ func getSlsaEntity(name string, digests scommon.DigestSet) (*slsaEntity, error)
 		},
 	}
 
-	if pkg, err := helpers.PurlToPkg(name); err == nil {
+	if name == "" {
```

Comment:
can you add a unit test to test this functionality with a new slsa attestation that has the URI set?

Reply:
Hey should I add a new predicate or modify the existing slsa predicate.

Reply:
can add a new one for testing.